src/module/system/damage/roll.ts

```diff
@@ -485,6 +488,7 @@ class DamageInstance extends AbstractDamageRoll {
                 `damage:type:${this.type}`,
                 typeCategory ? `damage:category:${typeCategory}` : [],
                 this.persistent ? "damage:category:persistent" : [],
+                this.splash ? "damage:component:splash" : [],
```

Comment:
This is marking the entire instance as splash damage.

Reply:
Just some question so i understand: 
Is splash damage it's own damage instance?
![ksnip_20250219-210642](https://github.com/user-attachments/assets/eb160a23-2dcb-42c4-9449-17eb2adddcac) so that should mean in this example are 4 separate damage instances?

English is not my mother tongue so i might interpret the splash trait wrong; i interpret the splash trait as merging the damage and the splash together as one instance.